### PR TITLE
Add package: lf

### PR DESCRIPTION
**Pull request by gbyl**
I've been using [lf](https://github.com/gokcehan/lf) for a while and thought others might find it useful as well. Tested on rM1 and rM2.

### CODE REVIEW

File: package/lf/package [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+section="utils"
+maintainer="gbyl <gbyl@users.noreply.github.com>"
+license=MIT
+
+image=golang:v2.3
+source=("https://github.com/gokcehan/lf/archive/refs/tags/r27.zip")
+sha256sums=(a4f7b3ada4aa1348b7f102374d8580b6992977f7e84053aa04ef6aadb69dc205)
+
+build() {
+    # export GOARCH=arm64
```
Comment (Eeems) [inline package/lf/package]: If this isn't needed it should be removed

**Review by Eeems — CHANGES_REQUESTED** [review]

**Comment by Eeems** [comment]
image
I get this error when attempting to run it on my rM1.

File: package/lf/package (hunk 1 above)
Comment (gbyl) [reply package/lf/package]: I mistankenly pushed an earlier version of this recipe. I'll submit the fix now.

**Comment by gbyl** [comment]
Thank you for the review @Eeems. I forgot to push my last commit. It should work now.

**Review by Eeems — APPROVED** [review]
Looks good to me, it works on my rM1

**Pull request merged**